browser.js

```diff
@@ -255,7 +255,7 @@ Requests.type = 'XHR' === Requests.method ? (function detect() {
     type = types.pop();
 
     xhr = Requests.XHR();
-    xhr.open('get', '/', true);
+    xhr.open('get', 'http://localhost/', true);
```

Comment:
I get issues with JSDOM's XHR (specifically, its URL parser) if a full URL is not used. Even then, xhr.open() doesn't actually open any connections; it just initializes the request object. Example to demonstrate using `netcat`:

```
$ nc -v -l localhost 8080
Listening  on [localhost] (family 0, port 8080)
```

In browser console (tested in Firefox 54.0):
```javascript
var req = new XMLHttpRequest();
req.open('GET', 'http://localhost:8080/', true);
```

Back in the shell, nothing should be printed. However, after:
```javascript
req.send()
```

`netcat` outputs something like:
```
Connection from [127.0.0.1] port 8080 [tcp/http-alt] accepted (family 2, sport 51196)
GET / HTTP/1.1
Host: localhost:8080
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:54.0) Gecko/20100101 Firefox/54.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Origin: null
DNT: 1
Connection: keep-alive

```

Regardless of the above, I wonder if there's any way around this URL parsing issue that doesn't involve changing the URL into a full one... thoughts? Perhaps this is an issue with JSDOM?